### PR TITLE
GitHub Actions with docker-images and smoketest

### PR DESCRIPTION
**Pull request by cfstras**
Runs the server and performs a ping test.

Also fixed some paths that were wrong due to the different docker image.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -1,14 +1,19 @@
 version: "3"
 services:
+
   mumble:
-    build: mumble/.
+    image: docker.pkg.github.com/flipdot/mumble-docker/mumble:latest
+    command: /usr/bin/murmurd -v -fg -ini /etc/murmur/murmur.ini
```
Comment (0boro) [inline docker-compose.yaml]: why defined it here, when we have CMD in the Dockerfile?

File: docker-compose.yaml [hunk 2]
```diff
@@ -1,14 +1,19 @@
 version: "3"
 services:
+
   mumble:
```
Comment (0boro) [inline docker-compose.yaml:4]: Please solve murmur/mumble complete ;)

File: docker-compose.yaml [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
-    build: mumble/.
+    image: docker.pkg.github.com/flipdot/mumble-docker/mumble:latest
```
Comment (0boro) [inline docker-compose.yaml:5]: Please solve murmur/mumble complete ;)

File: .github/workflows/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build_docker_images:
+    strategy:
+      matrix:
+        image_name:
+          - mumble
```
Comment (0boro) [inline .github/workflows/docker-compose.yml:15]: Please solve murmur/mumble complete ;)

File: .github/workflows/docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+          -newkey rsa:4096 \
+          -nodes \
+          -keyout data/privkey.pem \
+          -out data/fullchain.pem \
+          -days 30 \
+          -subj '/CN=localhost'
+
+    - name: Modify docker-compose.yaml
+      run: |
+        sed -i "s#docker\.pkg\.github\.com/.*mumble:latest#$(cat image_mumble/image.txt)#" docker-compose.yaml
```
Comment (0boro) [inline .github/workflows/docker-compose.yml:104]: Please solve murmur/mumble complete ;)

File: plugins/botamusique-docker/configuration.ini [hunk 6]
```diff
@@ -1,5 +1,5 @@
 [server]
-host = mumble.flipdot.org
+host = mumble
```
Comment (0boro) [inline plugins/botamusique-docker/configuration.ini:2]: Please solve murmur/mumble complete ;)

**Review by 0boro — CHANGES_REQUESTED** [review]
- when you want to use murmur instead mumble as naming for the mumble server component, use it consequent

File: docker-compose.yaml (hunk 1 above)
Comment (cfstras) [reply docker-compose.yaml]: added it for the "supw" flag which isn't needed...

But i'll replace all the murmurs with mumble, so we'll need it again :)

Comment (cfstras) [reply docker-compose.yaml]: ok done

**Review by 0boro — APPROVED** [review]

**Pull request merged**